packages/fxa-graphql-api/src/backend/complexity.plugin.ts

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import { ApolloServerPlugin, GraphQLRequestListener } from '@apollo/server';
+import { Plugin } from '@nestjs/apollo';
+import { getComplexity, simpleEstimator } from 'graphql-query-complexity';
+import { GraphQLError } from 'graphql';
+import { GraphQLSchemaHost } from '@nestjs/graphql';
+import { Inject } from '@nestjs/common';
+import { MozLoggerService } from 'fxa-shared/nestjs/logger/logger.service';
+
+const maxComplexity = 1000;
+
+/**
+ * This plugin needs access to NestJS' injectable graphql schema host
```

Comment:
I only saw one other file named like this, LMK if a different name or directory placement is preferred.